prow/prstatus/prstatus.go

```diff
@@ -168,20 +169,44 @@ func (da *DashboardAgent) HandlePrStatus(queryHandler PullRequestQueryHandler) h
 		data := UserData{
 			Login: false,
 		}
+
+		login := ""
 		if ok && token.Valid() {
-			data.Login = true
-			ghc := github.NewClient(token.AccessToken, githubEndpoint)
-			// Get user login
+			// If access token exist, get user login using the access token. This is a chance
+			// to validate whether the access token is consumable or not. If not, invalidate the
+			// session.
 			grc := pullRequestRestfulClient{ghclient.NewClient(token.AccessToken, false)}
-			login, err := getUserLogin(session, grc)
+			var err error
```

Comment:
why is this line necessary?

Reply:
Otherwise the `:=` below makes it less clear what is being shadowed?

Reply:
Ah, missed the scope of login.